server/proxy/proxy.go

```diff
@@ -190,6 +195,101 @@ func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath stri
 	return upstream.Notify(up, creq)
 }
 
+func (p *Proxy) NotifyMessageHasBeenPosted(post *model.Post, cc *apps.Context) error {
```

Comment:
I would say this function name is misleading. We are not notifying whether a message has been posted. We are notifying a bot mention.

Reply:
I think the intention is for this function to handle the other notification as well, eventually?

Reply:
@larkox @levb The function actually handles both the standard `created_post` subscription and `bot_mention` subscription. The name is meant to be a one-to-one match with the plugin hook, but prefixed with `Notify` to show that it is not the hook itself.